### PR TITLE
Disables advanced shuttle navigation consoles during nuclear war ops

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
:cl: Naksu
balance: The white ship and miscellaneous caravan ships lose their advanced place-anywhere shuttle movement during war ops.
/:cl:

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
cobby....was....here?

File: code/modules/shuttle/navigation_computer.dm [hunk 1]
```diff
@@ -26,6 +27,12 @@
 	. = ..()
 	GLOB.navigation_computers -= src
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/attack_hand(mob/user)
+	if(jammed)
+		to_chat(user, "<span class='warning'>The Syndicate are jamming the console!</span>")
```
Comment (AlexTheSysop42) [inline code/modules/shuttle/navigation_computer.dm]: is

**Review by xenon-difluoride — CHANGES_REQUESTED** [review]

**Comment by Buggy123** [comment]
Well, at least its atomic.

**Comment by SgtxRootbeer** [comment]
Ok, now we can say this is a bad idea.

**Comment by Buggy123** [comment]
>Ok, now we can say this is a bad idea.

**Comment by Tlaltecuhtli** [comment]
i ded

**Comment by WJohn** [comment]
I'm going to reiterate what I already said in buggy's closed PR and in discord but to sum it up: This is a very lazy way to go about making it only slightly harder to find their ship. It doesn't fix the fact that stealing the ship or the nuke invalidates the entire game mode, and THOSE are the issues that need to be dealt with instead.

Here's some fixes for that instead:
1. The nuke automatically teleports back to its place in the syndie ship if moved off the z level or destroyed.
2. The syndie pinpointer is changed so it can toggle modes to one of four options (off, nuke disk, syndie nuke, syndie ship).
3. Nuke op's code is identical to the station's nuke, so it is possible as a backup to detonate the station's nuke if yours is compromised.
4. Projectile mechanics are changed so they can actually hit people nearby if they're fired from inside a wall like with turrets, because right now you can hang around right next to a turret (a tile beneath it) and you're completely, totally safe. If that's not an option, maybe make them fire special high damage x-ray lasers.
5. The syndie ship cannot be sent home until the nuclear bomb is planted, or someone recalls it from the base. This is so people can't hijack it and hold the nuke hostage in a location the syndies can't possibly get to, which is the same as spacing or destroying the nuke. Recalling using the recall console on the base should stick around in case an op is left behind on the base.

**Comment by Buggy123** [comment]
Remove #4 in there, Kor has made it clear that he does not want x-ray turrets: #36238

**Comment by AlexTheSysop42** [comment]
>5. The syndie ship cannot be sent home until the nuclear bomb is planted, or someone recalls it from the base. This is so people can't hijack it and hold the nuke hostage in a location the syndies can't possibly get to, which is the same as spacing or destroying the nuke. Recalling using the recall console on the base should stick around in case an op is left behind on the base.

Your original idea of giving them a way to call the shuttle back to the station z-level would be better in case they somehow need to leave without the nuke being armed, while still keeping the crew from denying them the nuke

**Comment by AlexTheSysop42** [comment]
also with 4 the main idea is removing the safezone

**Comment by WJohn** [comment]
Oh, sorry I copy pasted this from a while ago. Turrets used to fucking suck at hitting things. Ignore number 4, it's already done.

**Comment by ExcessiveUseOfCobblestone** [comment]
5 can't work, there's an explicit ending where you nuke the syndicate base (super lose)

**Comment by AlexTheSysop42** [comment]
you can arm the nuke on the shuttle I think

**Comment by WJohn** [comment]
If you can arm the nuke on the shuttle, and send the shuttle back to their base, this would still be perfectly doable.

**Comment by AlexTheSysop42** [comment]
I was a nukie a few rounds ago and (even though I died horribly first) the crew found the ship and stole the nuke. It was parked just outside of arrivals, but even so, it's too easy as the crew to insta-win just because you space the nuke. This is after the recent updates in #36215.

**Comment by YakumoChen** [comment]
don't park the ship so close to the station. It's literally against the rules for the crew to go shuttle-hunting, but if you part it right outside, the rule isn't going to matter much any more anyway.

**Comment by SgtxRootbeer** [comment]
Kor you smoothbrain. This is a code solution to a player problem and aside from adding a nice hard tell, you're also acting like robustin, (blowing up the armory for the sake of a shitty gamemode)


CEASE

**Comment by AlexTheSysop42** [comment]
it only trigger during war, but this still doesnt keep the crew from spacing the nuke

**Comment by KorPhaeron** [comment]
A hard tell for the game mode that has a centcomm announcement roundstart oh no

**Comment by SgtxRootbeer** [comment]
>during nuclear war ops
Oh god I'm autistic.

**Comment by adamkad1** [comment]
so, are you gonna nerf syndie ship turrets or just keep buffing syndies? its not like whiteship can compete with syndieship

**Comment by adamkad1** [comment]
> don't park the ship so close to the station. It's literally against the rules for the crew to go shuttle-hunting, but if you part it right outside, the rule isn't going to matter much any more anyway.

 not since they added those dumb turrets

**Comment by YakumoChen** [comment]
@adamkad1 you know this is a PR from literally last year right? are you just going back and leaving stupid comments on every PR?

**Comment by ExcessiveUseOfCobblestone** [comment]
frick off

**Pull request merged**